app/Policies/NewsPolicy.php

```diff
@@ -37,27 +39,43 @@ public function view(?User $user, News $news): bool
     public function create(User $user): bool
     {
         return $user->hasAnyRole([
+            Role::ADMINISTRATOR,
+            Role::DEVELOPER_STAFF,
+            Role::DEVELOPER,
+            Role::EVENT_MANAGER,
+            Role::MODERATOR,
             Role::NEWS_MANAGER,
+            Role::TEAM_ACCOUNT,
         ]);
     }
 
     public function update(User $user, News $news): bool
     {
         return $user->hasAnyRole([
+            Role::ADMINISTRATOR,
+            Role::DEVELOPER_STAFF,
+            Role::DEVELOPER,
+            Role::EVENT_MANAGER,
+            Role::MODERATOR,
             Role::NEWS_MANAGER,
+            Role::TEAM_ACCOUNT,
```

Comment:
This should maybe be limited to NEWS_MANAGER and MODERATOR unless the news item was created by the current user.

Reply:
Same comment as above -- I agree, this is only in place to preserve existing functionality until the legacy page is deleted.